solidity/contracts/BancorNetwork.sol

```diff
@@ -37,6 +37,7 @@ contract BancorNetwork is IBancorNetwork, TokenHolder, ContractRegistryClient, F
 
     uint256 private constant CONVERSION_FEE_RESOLUTION = 1000000;
     uint256 private constant AFFILIATE_FEE_RESOLUTION = 1000000;
+    IEtherToken private constant ETH_RESERVE = IEtherToken(-1);

```

Comment:
maybe use something more specific like `ETH_RESERVE_ADDRESS` or `ETH_ADDRESS`?
also address should be 0xeeee.... and not 0xffff....

Reply:
@yudilevi : Please see updated code.